_includes/configuration.md

```diff
@@ -1,152 +1,47 @@
 
-6.  Press `Enter` to accept a setting where applicable.
+6. Press `Enter` to accept a setting where applicable.
 
 **Note:**
 
-The {{ site.data.product.title_short }} appliance console automatically logs out
-after five minutes of inactivity.
-
-### Configuring a Database
-
-{{ site.data.product.title_short }} uses a database to store information about the
-environment. Before using {{ site.data.product.title_short }}, configure the database
-options for it; {{ site.data.product.title_short }} provides the following two
-options for database configuration:
-
-  - Install an internal PostgreSQL database to the appliance
-
-  - Configure the appliance to use an external PostgreSQL database
-
-#### Configuring an Internal Database
-
-{% include configuration-db.md %}
-
-#### Configuring an External Database
-
-Based on your setup, you will choose to configure the appliance to use
-an external PostgreSQL database. For example, we can only have one
-database in a single region. However, a region can be segmented into
-multiple zones, such as database zone, user interface zone, and
-reporting zone, where each zone provides a specific function. The
-appliances in these zones must be configured to use an external
-database.
-
-The `postgresql.conf` file used with {{ site.data.product.title_short }} databases requires
-specific settings for correct operation. For example, it must correctly
-reclaim table space, control session timeouts, and format the PostgreSQL
-server log for improved system support. Due to these requirements, Red
-Hat recommends that external {{ site.data.product.title_short }} databases use a
-`postgresql.conf` file based on the standard file used by the
-{{ site.data.product.title_short }} appliance.
-
-Ensure you configure the settings in the `postgresql.conf` to suit your
-system. For example, customize the `shared_buffers` setting according to
-the amount of real storage available in the external system hosting the
-PostgreSQL instance. In addition, depending on the aggregate number of
-appliances expected to connect to the PostgreSQL instance, it may be
-necessary to alter the `max_connections` setting.
-
-**Note:**
-
-  - {{ site.data.product.title_short }} requires PostgreSQL version 9.5.
-
-  - Because the `postgresql.conf` file controls the operation of all
-    databases managed by a single instance of PostgreSQL, do not mix
-    {{ site.data.product.title_short }} databases with other types of databases in a single
-    PostgreSQL instance.
-
-1.  Start the appliance and open a terminal console.
-
-2.  Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance
-    summary screen displays.
-
-3.  Press **Enter** to manually configure settings.
-
-4.  Select **Configure Application** from the menu.
-
-5.  You are prompted to create or fetch a security key.
-
-      - If this is the first {{ site.data.product.title_short }} appliance, choose **Create
-        key**.
```

Comment:
This line should probably be removed.  By default a key should have already been generated by this point.

Reply:
Actually this whole number 5 is confusing...I don't know why we'd need this at all if the key already exists...I'll have to check on a real appliance.